src/Products/Products.php

```diff
@@ -39,28 +39,106 @@ public function delete(string $id) {
     }
 
     public function index(){

```

Comment:
ojej....
Za dużo zagnieżdżeń, za dużo elssifów. 

Takie przekładańce załatwia "early return".

Przykład:
teraz jest:

```php
if($this->checkAlertIsAccepted == true)
{
// do sth 1
}elseif($this->checkAlertIs Accepted==false)
{
// do sth 2
}
```

można to zrobić tak:

```php
if($this->checkAlertIsAccepted===false)
{
// do sth 2
return; 
}

// do sth 1 ( domyślne zachowanie )
```